jscomp/syntax/tests/parsing/grammar/expressions/expected/UncurriedByDefault.res.txt

```diff
@@ -171,4 +171,6 @@ type nonrec callback1 =
 type nonrec callback2 =
   (ReactEvent.Mouse.t -> unit as 'u, [ `Has_arity1 ]) function$
 type nonrec callback3 =
-  (ReactEvent.Mouse.t -> unit, [ `Has_arity1 ]) function$ as 'callback
\ No newline at end of file
+  (ReactEvent.Mouse.t -> unit, [ `Has_arity1 ]) function$ as 'callback
+let partialPipe = x |.u ((f ())[@res.partial ][@res.uapp ])
```

Comment:
Not sure about the `()`. Try an example and see if it type checks correctly.

Reply:
Indeed, it shouldn't be there. But if I remove it, the above end up as:

```rescript
let partialPipe = x |.u f
```

Seems the whole `apply` expression is replaced by just the function identifier, which makes  sense in itself of course, but that also means the attributes are dropped. Might not be so easy to fix this one.

Reply:
Yes looks like fixing this one is more complicated and might need to involve the processing of the partial attribute later in the compiler.
E.g. conceptually I guess the attribute belongs to `|.u`  in that example. Or perhaps putting it on `f` is fine.
Guess it does not matter much -- just some representation picked up deeper down in the compiler.